Automattic/wp-calypso
# PlansSkipButton: Move to Components

(The 'Start with Free' button at the bottom of the Jetpack Plans page:)

![image](https://user-images.githubusercontent.com/96308/38953331-b7230dc4-434e-11e8-82da-75f20570f557.png)

So we can re-use it for WP.com Plans (in a subsequent PR). While we're at it, also update tests to use enzyme's `shallow` rendering instead of `react-test-renderer` (which we should get rid of altogetther).

### Testing Instructions

* Create a new JP site, and connect it to WP.com
* When reaching the Plans page, change the `wordpress.com` in the URL to `calypso.localhost:3000` and load that page.
* Verify that the 'Start with Free' button still works
* Grep to verify that all occurrences of `plans-skip-button` have been handled.
